classes/output/core_renderer.php

```diff
@@ -960,6 +960,8 @@ private function get_login() {
             return false;
         }
 
+        return ['url' => get_login_url()];
```

Comment:
this looks wrong :D is this a leftover from debugging?

Reply:
Ups ^^